CONTRIBUTING.md

```diff
@@ -1,82 +1,79 @@
+To clone the repository, you must clone the repository and its submodules using the recursive flag:
 
 ```
 git clone --recursive https://github.com/LawnchairLauncher/lawnchair.git
 ```
 
-To build the Lawnchair Launcher, select the `lawnWithQuickstepDebug` build type. Should you face errors relating to the `iconloaderlib` and `searchuilib` projects, run `git submodule update --init --recursive`.
+To build `Lawnchair Launcher`, select the `lawnWithQuickstepDebug` build type. Should you face errors relating to the `iconloaderlib` and `searchuilib` projects, run `git submodule update --init --recursive`.
 
 Here are some contribution tips to help you get started:
 
-- Always make sure that you're up-to-date with Lawnchair Launcher by setting your branch to `14-dev` to newer;
-  
-- Make sure your code is logical and well-formatted. If using Kotlin, see [“Coding conventions” in the Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
-  
-- [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum
-  
+-   Always make sure that you're up-to-date with `Lawnchair Launcher` by setting your branch to `14-dev` or newer;
+
+-   Make sure your code is logical and well-formatted. If using Kotlin, see `Coding conventions` in the [Kotlin documentation](https://kotlinlang.org/docs/coding-conventions.html);
+
+-   [The `lawnchair` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/lawnchair) houses Lawnchair’s own code, whereas [the `src` package](https://github.com/LawnchairLauncher/lawnchair/tree/14-dev/src) includes a clone of the Launcher3 codebase with modifications. Generally, place new files in the former, keeping changes to the latter to a minimum.
 
 #### Version
 
-As of [#4361](https://github.com/LawnchairLauncher/lawnchair/pull/4361), the Lawnchair Launcher version code is separated by four sectors:
+As of [#4361](https://github.com/LawnchairLauncher/lawnchair/pull/4361), the `Lawnchair Launcher` version code is separated by four sectors:
+
+1. `Android`'s major version
+
+2. `Android`'s minor version
+
+3. `Lawnchair Launcher`'s development status
 
-1. Android's Major version
-  
-2. Android's Minor version
-  
-3. Lawnchair Launcher's development status
-  
-4. Lawnchair Launcher's development version
-  
+4. `Lawnchair Launcher`'s development version
 
-##### Android's Major & Minor version
+##### Android's major & minor versions
 
-These makes up the first two sectors of the version code, Android 11 will be `11_00_XX_XX` while Android 12.1 will be `12_01_XX_XX`.
+These makes up the first two sectors of the version code, `Android 11` will be `11_00_XX_XX` while `Android 12.1` will be `12_01_XX_XX`.
```

Comment:
Android 11 and 12.1 refer to the Operating System, Android, therefore should not be marked as codeblock here.

```suggestion
These makes up the first two sectors of the version code, Android 11 will be `11_00_XX_XX` while Android 12.1 will be `12_01_XX_XX`.
```